html/modules/custom/reliefweb_post_api/schemas/v2/report.json

```diff
@@ -242,23 +242,23 @@
             "required": ["url", "uuid", "checksum", "description", "copyright"]
         },
         "disaster": {
-            "description": "Document disaster(s) as a list of IDs from https://api.reliefweb.int/v1/disasters.",
+            "description": "Document disaster(s) as a list of IDs from https://api.reliefweb.int/v1/disasters. Only select the most relevant terms for the document. Over-tagging often results in reduced discoverability.",
```

Comment:
"Only select the most relevant terms for the document." is only added here for the "disaster" field. Should this sentence be removed or added to the other taxonomy fields for consistency?

Reply:
I think all the taxonomy terms that might be over-tagged are now covered